liaisons/opcf/tech_reqs.md

```diff
@@ -0,0 +1,23 @@
+# Technical Requirements for OPC UA/WoT Collaboration
+
+## Use Cases
+See https://w3c.github.io/wot-usecases/#use-case-opc-ua-binding
+
+## Objectives
+1. Use OPC UA servers (device, node) from WoT Consumers; treat an OPC UA server as a Thing
+2. Access WoT TD metadata for OPC UA servers - integrate discovery process with OPC UA
+3. ? Let OPC UA servers connect to WoT Things; but if OPC UA devices are ALSO WoT Consumers, 1 satisfies this; --> "OPC UA Servients"
+
+## Deliverables
+1. OPC UA Ontology
+   - Defines information model and vocabulary for all OPC UA definitions and protocol options used in UA NodesetFile
+   - RDF/TTL/JSONLD Context file and URL, suitable for use in JSON-LD and RDF
+   - Note: useful for activites other than 2
+2. WoT TD Protocol Binding for OPC UA Nodesets
+   - Depends on 1 (TDs with this protocol binding would include the OPC UA ontology defined in 1)
+3. Process to transform a OPC UA NodesetFile to a WoT Thing Description (and vice versa, if TD describes OPC UA Nodeset)
+   - May have to infer or convert certain things to do conversion
+   - May have to gather information from places other than NodesetFile
```

Comment:
Which places? An example would help to understand better.

Reply:
Also makes me wonder about dynamic nodes - i.e. nodes that aren't included in the static nodeset definitions ...

We have a few instances of interfaces like that...

Reply:
Well, I was being intentionally vague here because I don't know what information might be needed in a TD that is not in a Nodeset file or where it might come from.  At the time I wrote this however I was thinking about information that might be available only in human-readable documents, like dataschemas.  I'm not sure exactly how Nodeset files are organized, but if assumptions are made about, say, message formats, we probably need to look at the actual specifications to model the data schemas for such messages, as the information may not be explicit in the Nodeset file in a form we can automatically translate.  However... to resolve this issue, let me add a "such as" clause.
```suggestion
   - May have to gather information from places other than NodesetFile, such as human-readable specifications.
```